src/js/data/cosmos/index.ts

```diff
@@ -0,0 +1,79 @@
+import { of, throwError } from "rxjs";
+import { map, retry, switchMap } from "rxjs/operators";
+import { makeExecutableSchema } from "graphql-tools";
+import { RequestResponse } from "@dcos/http-service";
+import { CosmosClient, PackageVersionsResponse } from "cosmos-client";
+
+import Config from "#SRC/js/config/Config";
+import { Package, PackageSchema } from "#SRC/js/data/cosmos/Package";
+import { PackageVersionSchema } from "#SRC/js/data/cosmos/PackageVersion";
+
+export interface ResolverArgs {
+  cosmosClient: CosmosClient;
+}
+
+export interface GeneralArgs {
+  [key: string]: any;
```

Comment:
Just curious, why this instead of just `object` type?

Reply:
At least for me `object` will fail tslint in my editor when I try to access any key.

Reply:
Would a union of all "specificArgs" do? So maybe `export type GeneralArgs = Partial< PackageQueryArgs >;`?